### PR TITLE
UI: Moved Backlog

### PR DESCRIPTION
**Pull request by Amstutz**
With the introduction of the Roadmap, I do not see the purpose of the Backlog anymore. Most of the items there are outdated anyways. I removed the backlog and updated the roadmap slightly. @klees Please browse through I check if I missed anything.

### CODE REVIEW

File: src/UI/ROADMAP.md [hunk 1]
```diff
@@ -43,6 +43,19 @@ test for all existing examples, i.e. checks if the example can be executed at
...
 
+### Examples on Main Page (Beginner, ~4h)
+
+We want to have examples on the main pages of some components family of the 
+documentation displayed in ILIAS System Styles Section. E.g. there should
+also be examples on the "Buttons" Page for the complete Buttons family.
+
+
+Create an abstraction for Actions that could be used instead of stringy links.
+  It would be strongly related to ilCtrl and probably should be an ILIAS library.
```
Comment (klees) [inline src/UI/ROADMAP.md]: This is missing a headline. This IMO also fits better in the ideas section, where also no headline would be required.

File: src/UI/ROADMAP.md [hunk 2]
```diff
@@ -43,6 +43,19 @@ test for all existing examples, i.e. checks if the example can be executed at
...
+  
+### Automatic Less-Variables Extraction (Expert, ~2PD)
+Create a mechanism to wire less-files to delos.less that is more automatic than
+'do it manually'.
```
Comment (klees) [inline src/UI/ROADMAP.md]: I would put this to the ideas section as well, since I currently see no concrete plan of execution here.

**Review by klees — COMMENTED** [review]
Thx a lot!

**Comment by Amstutz** [comment]
Thx for the feedback. Updated it.

File: src/UI/ROADMAP.md [hunk 3]
```diff
@@ -43,7 +43,16 @@ test for all existing examples, i.e. checks if the example can be executed at
...
 
+### Examples on Main Page (Beginner, ~4h)
 
+We want to have examples on the main pages of some components family of the 
+documentation displayed in ILIAS System Styles Section. E.g. there should
+also be examples on the "Buttons" Page for the complete Buttons family.
+
+### Abstraction for Actions (advanced, ~8h)
+Create an abstraction for Actions that could be used instead of stringy links.
+It would be strongly related to ilCtrl and probably should be an ILIAS library.
```
Comment (klees) [inline src/UI/ROADMAP.md]: Is this really actionable? I currently cannot see how this fits into the bigger picture of signal/triggerer and what this needs to be (besides some wrapper around string). But maybe I'm thinking too big.

Comment (Amstutz) [reply src/UI/ROADMAP.md]: I just copied your point from the roadmap. If not needed anymore, just kick it.

**Pull request merged**